schema.sql

```diff
@@ -1,25 +1,27 @@
+DROP SCHEMA IF EXISTS trelia CASCADE;
 
 CREATE SCHEMA trelia
-  CREATE TABLE listings (
-    listing_id serial PRIMARY KEY,
+  CREATE TABLE trelia.listings (
+    listing_id integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
interesting, I'm going to look more into this

Reply:
there's a good discussion about serial vs identity data types in Chris's PR that Amelia reviewed (link below)

[](https://github.com/hrsf131-group3/Main-Gallery/pull/2)